text/0061-error-format-and-definitions.md

```diff
@@ -578,163 +1038,674 @@ This error occurs when the user specifies a non-existent ranking rule, a malform
-    ...
+    "message": "`:fieldName` setting is invalid. //TODO",
+    "code": "invalid_settings_typo_tolerance",
+    "type": "invalid_request",
+    "link": "https://docs.meilisearch.com/errors#invalid_settings_typo_tolerance"
 }
 ```
 
-- The `:reservedKeyword` is inferred when the message is generated.
+- The `:fieldName` is inferred when the message is generated.
 
-#### Variant: Using `_geo` or `_geoPoint` as a filter expression
+#### Variant: `minWordSizeForTypos` object of the `typo` resource is invalid. (//TODO: make it synchronous)
 
 ```json
 {
-    "message": "`:reservedKeyword` is a reserved keyword and thus can't be used as a filter expression. Use the _geoRadius(latitude, longitude, distance) built-in rule to filter on _geo field coordinates.",
-    ...
+    "message": "`minWordSizeForTypos` setting is invalid. `oneTypo` and `twoTypos` fields should be between `0` and `255`, and `twoTypos` should be greater or equals to `oneTypo` but found `oneTypo: :oneTypo` and twoTypos: twoTypos`.",
 }
 ```
 
-- The `:reservedKeyword` is inferred when the message is generated.
+- The `:oneTypo` is inferred when the message is generated.
+- The `:twoTypos` is inferred when the message is generated.
 
-#### Variant: Invalid syntax for the `filter` parameter
+---
+
+## invalid_settings_faceting
+
+`Synchronous`
+
+### Context
+
+This error occurs when a value different from `null` or with a different type than `Integer` is specified for the `maxValuesPerFacet` field.
+
+### Error Definition
+
+HTTP Code: `400 Bad Request`
 
 ```json
 {
-    "message": "Invalid syntax for the filter parameter: `:syntaxErrorHelper`.",
-    ...
+    "message": "//TODO",
+    "code": "invalid_settings_faceting",
+    "type": "invalid_request",
+    "link": "https://docs.meilisearch.com/errors#invalid_settings_faceting"
 }
 ```
 
-- The `:syntaxErrorHelper` is inferred when the message is generated.
+---
+
+## invalid_settings_pagination
+
+`Synchronous`
+
+### Context
+
+This error occurs when a value different from `null` or with a different type than `Integer` is specified for the `maxTotalHits` field.
+
+### Error Definition
+
+HTTP Code: `400 Bad Request`
+
+```json
+{
+    "message": "//TODO",
+    "code": "invalid_settings_pagination",
+    "type": "invalid_request",
+    "link": "https://docs.meilisearch.com/errors#invalid_settings_pagination"
+}
+```
 
 ---
 
-## invalid_sort
+## invalid_search_filter
 
 `Synchronous`
 
 ### Context
 
-This error occurs at search time when there is a syntax error in the `sort` parameter, when an attribute expressed in the sort is not defined in the `sortableAttributes` list, sort at search time while the `sort` ranking rule is missing from the settings, or using reserved keywords like `_geo`, `_geoDistance` and `_geoRadius` as a sort expression.
+This error occurs when:
 
-### Error Definition:
+- there is a syntax error in the `filter` parameter
```

Comment:
Maybe those statements should start with a capital letter and have a full stop in the end?